src/databricks/labs/ucx/install.py

```diff
@@ -676,17 +675,39 @@ def _get_workspace(workspace_id: int, ids_to_workspace: dict[int, Workspace]) ->
             msg = "Current workspace is not known, Please run as account-admin: databricks labs ucx sync-workspace-info"
             raise KeyError(msg) from None
 
-    def _get_workspace_info(self, current_workspace_id: int):
+    def _get_workspace_ids(self):
         account_client = self._get_safe_account_client()
-        workspace = account_client.workspaces.get(current_workspace_id)
-        current_workspace_client = account_client.get_workspace_client(workspace)
-        installation = Installation.current(current_workspace_client, self.product_info.product_name())
-        workspace_info = WorkspaceInfo(installation, current_workspace_client)
-        return workspace_info.load_workspace_info()
+        workspaces = account_client.workspaces.list()
```

Comment:
looks like a duplicate logic to ctx.account_workspaces.get_accessible_workspaces()

Reply:
updated